packages/app-root/src/app/about/team/response.mock.json

```diff
@@ -0,0 +1,39 @@
+[
+  {
+    "name": "Alumni",
+    "weight": 100,
+    "people": [
+      {
+        "bio": "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
+        "jobTitle": "Project Assistant",
+        "name": "Mock Person One",
+        "avatarSrc": "https://placekitten.com/300/300?w=160&h=160"
```

Comment:
When I run `yarn build` then `yarn start` and check https://local.zooniverse.org:3000/about/team the avatar images are not displayed.

Reply:
placekitten.com has been out of commission for awhile 😢 I'll choose a different url